docs/user/SingleStore.md

```diff
@@ -1,27 +1,33 @@
+You can deploy SAS Viya with SingleStore into a Kubernetes cluster that is running in Microsoft Azure. The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an Azure cluster suitable for deploying SAS Viya and SingleStore.
 
 ## Customizing SingleStore Deployment Overlays
 
-Refer to the SAS Viya Deployment instructions [Getting Started](https://github.com/sassoftware/viya4-deployment#getting-started) and the [SAS Viya Customizations](https://github.com/sassoftware/viya4-deployment#sas-viya-customizations) section for information on how users can make changes to their deployment by adding custom overlays into subfolders under the `/site-config` folder.
+Refer to the viya4-deployment [Getting Started](https://github.com/sassoftware/viya4-deployment#getting-started) and [SAS Viya Customizations](https://github.com/sassoftware/viya4-deployment#sas-viya-customizations) documentation if you need information about how to make changes to your deployment by adding custom overlays into subdirectories under the `/site-config` directory.
+
+After running viya4-deployment with the setting `DEPLOY=false` in your ansible-vars.yaml file, locate the `sas-bases/` directory, which is a peer to the `site-config/` directory underneath your SAS deployment's <base_dir>.
+
+Complete each step under the **SingleStore Cluster Definition** heading in the `sas-bases/examples/sas-singlestore/README.md` file in order to configure your SAS Viya with SingleStore deployment, noting the following exceptions:
 
-After running SAS Viya Deployment with the setting DEPLOY=false in your ansible-vars.yaml file, locate the `sas-bases/` folder which is a peer to the `site-config/` folder underneath your deployments <base_dir>.
+- Add a new Step 1a after Step 1:
 
-Follow each step under the `SingleStore Cluster Definition` heading in the `sas-bases/examples/sas-singlestore/README.md` file to configure your SingleStore deployment noting the exceptions below:
+>>1a. Copy `$deploy/sas-bases/components/sas-singlestore` into the `$deploy/site-config/sas-singlestore/component` directory.
```

Comment:
I used `components` as my directory vs `component` seemed to work. Are we sure each product will always have one component?

Reply:
Yes, I think "`$deploy/site-config/sas-singlestore/component` directory" is better.